kordac/Kordac.py

```diff
@@ -60,9 +61,10 @@ def convert(self, text):
         self.kordac_extension.clear_saved_data()
         html_string = self.converter.convert(text)
         result = KordacResult(
-            html_string=html_string,
-            title=self.kordac_extension.title,
-            required_files=self.kordac_extension.required_files
+            html_string = html_string,
+            title = self.kordac_extension.title,
+            required_files = self.kordac_extension.required_files,
+            heading_tree = self.kordac_extension.get_heading_tree()
```

Comment:
@JackMorganNZ tends to use the style without spaces...should probably keep things consistent...or change ALL THE THINGS